framework/test/mocha/integration/transactions/exceptions/sigantures.js

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
should probably fix this file name

Reply:
it seems like test itself is checking the signatures exception.
Probably description needs to be changed

Reply:
oh I meant just the typo `sigantures` --> `signatures` :)